action.yml

```
@@ -110,9 +78,17 @@ runs:
       env:
         CHART: ${{ inputs.chart }}
         REMOTE: ${{ inputs.remote }}
-        API_TOKEN: ${{ cloudbees.api.token }}
-        URL: ${{ cloudbees.api.url }}
-        RUN_ID: ${{ cloudbees.run_id }}
-        RUN_ATTEMPT: ${{ cloudbees.run_attempt }}
         ARTIFACT_NAME: ${{ inputs.artifact-name }}
-        COMPONENT_ID: ${{ inputs.component-id }}
+    - id: registerArtifact
```

Lets use kebob-case as a convention for step id. 
```suggestion
    - id: register-artifact
```